pages/workspaces/new.tsx

```diff
@@ -10,6 +12,23 @@ import { TrackedReposTable } from "components/Workspaces/TrackedReposTable";
 import { createWorkspace } from "lib/utils/workspace-utils";
 import { WORKSPACE_UPDATED_EVENT } from "components/shared/AppSidebar/AppSidebar";
 
+export const getServerSideProps = async (context: GetServerSidePropsContext) => {
```

Comment:
Need to guard against someone unauthenticate trying to create a workspace. This wasn't an issue before because unauthenticated users couldn't hit this route.